host_core/lib/host_core/control_interface/server.ex

```diff
@@ -202,6 +202,21 @@ defmodule HostCore.ControlInterface.Server do
     {:reply, response}
   end
 
+  # Stop Host
+  defp handle_request({"cmd", _host_id, "stop"}, body, _reply_to) do
+    # TODO: Right now this will contain a parameter for timeout. Obviously how this works currently
+    # only results in the graceful shutdowns built into the system. There may be some inflight work
+    # we want to wait for up to the timeout. We could use this library possibly so we can put in
+    # hooks: https://github.com/botsquad/graceful_stop.
+    _stop_host_command = Jason.decode!(body)
+
+    Logger.info("Received stop request for host")
```

Comment:
I know this is probably not part of the original scope, but I think we should create a `com.wasmcloud.lattice.host_stopped` cloud event and emit this, wait a few milliseconds, then call purge. This gives the lattice notice that a host has been _properly_ terminated (e.g. didn't crash).

You can see an example of publishing the "actor stopped" event here: https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/actors/actor_module.ex#L371